test/boost/aws_error_injection_test.cc

```diff
@@ -39,14 +39,14 @@ static std::string get_address() {
 
 static s3::endpoint_config_ptr make_minio_config() {
     s3::endpoint_config cfg = {
-        .port = get_port(),
-        .use_https = false,
-        .aws = {{
-            .access_key_id = "foo",
-            .secret_access_key = "bar",
-            .session_token = "baz",
-            .region = "us-east-1",
-        }},
+            .port = get_port(),
```

Comment:
why? IMO, the existing indent looks better.

Reply:
formatted according to the .clang-format changes applied [here](https://github.com/scylladb/scylladb/pull/20968/files)

Reply:
nm, found a workaround for not being affected by changes made to `.clang-format` file ;)